src/mgmt/srv_pool.c

```diff
@@ -782,14 +782,37 @@ pool_get_ranks(struct mgmt_svc *svc, uuid_t uuid, d_rank_list_t **ranks)
 	return rc;
 }
 
+static int
+get_acl_for_pool(uuid_t pool_uuid, d_rank_list_t *ranks, struct daos_acl **acl)
+{
+	int			rc;
+	daos_prop_t		*prop;
+	struct daos_prop_entry	*entry;
+
+	rc = ds_pool_svc_get_acl_prop(pool_uuid, ranks, &prop);
```

Comment:
Is there a reason that this get is implemented in a acl specific way but the set is property generic?

Reply:
I'm actually planning to update that function to be more generic as part of DAOS-3535 (adding owner/group to get-acl output). It was a mistake to make it that specific - I think it should be very easy to make it generic.